desktop/core/src/desktop/lib/metrics/file_reporter.py

```diff
@@ -46,6 +46,7 @@ def report_now(self, registry=None, timestamp=None):
     # rename the file to the real location.
 
     f = tempfile.NamedTemporaryFile(
+        mode='w',
```

Comment:
import sys (with other imports)

'w' if sys.version_info[0] > 2 else 'w+b'

Reply:
Sure. That seems reasonable.